README.md

```diff
@@ -26,7 +26,7 @@ If you have a suggestion to improve an exercise, an idea for a new exercise, or
 4. Some of the exercises have test conditions defined in their spec file as `test.skip` compared to `test`. This is purposeful. After you pass one `test`, you will change the next `test.skip` to `test` and test your code again. You'll do this until all conditions are satisfied. **All tests must pass at the same time**, and you should not have any `test.skip` instances by the time you finish an exercise.
 5. Once you successfully finish an exercise, check the `solutions` directory within each exercise to compare it with yours.
    - You should not be checking the solution for an exercise until you finish it!
-   - Keep in mind that TOP's solution is not the only solution. Generally as long as all of the tests pass, your solution should be fine.
+   - If your solution differs wildly from TOP's solution (and still passes the self-check criteria), that's completely fine. Do feel free to ask about it in the chatroom if there are parts you do not understand.
```

Comment:
I feel like we need to note that the solution ideally should be in line with the concepts that were learned leading up to doing the exercise. For example, if an exercise is assigned after learning about string methods, ideally a user's solution should use string methods (vs loops). That may be a larger conversation/re-evaluating where each exercise is introduced within the context of the curriculum.

Unless you feel adding some verbiage like that would be helpful right now, I only have one nit below:

```suggestion
   - If your solution differs wildly from TOP's solution (and still passes the self-check criteria), that's completely fine. Do feel free to ask about it in our Discord if there are parts you do not understand.
```

Reply:
My feeling at the moment is that "meeting the exercise requirements" should be sufficient to imply that, depending on what the exercise requirements are. And that if there is an intention for an exercise to be done a certain way based on recent lesson content, then that would be better placed in the relevant exercise's requirements. Thoughts?

For now I'll make that nit change and also rephrase "self-check criteria" to "exercise's requirements", as there technically isn't a "self-check criteria" section in the exercises' READMEs.